udata/core/dataset/api_fields.py

```diff
@@ -26,9 +26,9 @@
 dataset_harvest_fields = api.model('HarvestDatasetMetadata', {
     'backend': fields.String(description='Harvest backend used', allow_null=True),
     'created_at': fields.ISODateTime(description='The dataset harvested creation date',
-                                     allow_null=True),
+                                     allow_null=True, readonly=True),
```

Comment:
Entire harvest fields are already set as readonly :)

Reply:
Do you think it's a problem to double down on the `readonly` here? In case we put the harvest field as non readonly somewhere, the `created_at` will always be `readonly`?